From 494f67212f570f6d16d5b2cd517b46d20872e1ab Mon Sep 17 00:00:00 2001
From: YangYonggang <b31664@freescale.com>
Date: Mon, 27 Feb 2012 15:29:41 +0800
Subject: [PATCH 243/635] ENGR00175464 Camera: Enable 1080p recorder.

In current driver, 1080p only support 15 fps. Changed the code to
enable 1080p recorder.

Signed-off-by: YangYonggang <b31664@freescale.com>
---
 mx6/libcamera/CameraHal.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mx6/libcamera/CameraHal.cpp b/mx6/libcamera/CameraHal.cpp
index d8754f9..6a55cbc 100755
--- a/mx6/libcamera/CameraHal.cpp
+++ b/mx6/libcamera/CameraHal.cpp
@@ -360,7 +360,7 @@ namespace android {
             }
             pictureCnt ++;
 
-            if (CaptureSizeFps.tv.denominator/CaptureSizeFps.tv.numerator > 25){
+            if (CaptureSizeFps.tv.denominator/CaptureSizeFps.tv.numerator >= 15){
                 if (previewCnt == 0)
                     strncpy((char*) mSupportedPreviewSizes, TmpStr, CAMER_PARAM_BUFFER_SIZE);
                 else{
-- 
1.8.0

